src/lib/components/SEO.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  export let description: string
+  export let image: string
+  export let imageAlt: string
+  export let ogType = 'website'
+  export let tags = ['vermouth', 'spansk', 'alkohol', 'premium']
+  export let title: string
+</script>
+
+<svelte:head>
+  <title>{title}</title>
+  <meta property="og:title" content={title} />
```

Comment:
There are missing meta tags - Facebook, iMessage, WhatsApp don't show image when sharing a link

Reply:
confirm that image shows on all pages when sharing in WhatsApp, Facebook, iMessage & LinkedIn